src/pow.cpp

```diff
@@ -226,7 +226,7 @@ RCP<const Basic> pow(const RCP<const Basic> &a, const RCP<const Basic> &b)
                 RCP<const Integer> exp_new = rcp_static_cast<const Integer>(a);
                 RCP<const Number> frac = exp_new->powint(Integer(q));
                 map_basic_basic surd;
-                if ((exp_new->is_negative()) && ((r / den) == 1/2)) {
```

Comment:
`r / den` is integer division and therefore `(-6)*(4/3)` is given as `-6*I*6^(1/3)`